subcommands/keys/tuf_updates_review.go

```diff
@@ -101,20 +103,50 @@ func doTufUpdatesReview(cmd *cobra.Command, args []string) {
 				}
 			}
 		}
+	} else if updates.Status == client.TufRootUpdatesStatusNone {
+		fmt.Println("There are no TUF root updates in progress.")
+		if len(updates.Advices.Mandatory) > 0 {
+			fmt.Println(`\nFoundries.io identified some problems with your existing TUF metadata.
+Please, consider these updates in order to keep your TUF metadata functional and secure:`)
+			for _, advice := range updates.Advices.Mandatory {
+				fmt.Printf(" - %s\n", advice.Message)
+			}
+		}
+		if len(updates.Advices.Recommended) > 0 {
+			fmt.Println("\nFoundries.io recommends these updates to improve your existing TUF metadata:")
+			for _, advice := range updates.Advices.Recommended {
+				fmt.Printf(" - %s\n", advice.Message)
+			}
+		}
+		if len(updates.Advices.Mandatory) > 0 || len(updates.Advices.Recommended) > 0 {
+			fmt.Println("\nPlease, also consider examples from 'fioctl keys tuf updates --help'.")
```

Comment:
I'm on the fence if this adds much value. We've just told them exactly what they need to do, and now we sort of tell them "BTW- please read the manual".

Reply:
I was on the fence here too; so it doesn't hurt me to remove it.

Reply:
Removed in force-push.